packages/react-internal/src/components/ComponentConformance.test.tsx

```diff
@@ -269,17 +38,6 @@ describe('Top Level Component File Conformance', () => {
     jest.resetModules();
   });
 
-  // Top Level Compoennt File Compliance -
-  // make sure that there is a corresponding top level component file for each component in the directory
-  components.forEach(componentName => {
-    it(`${componentName} has a corresponding top level component file`, () => {
-      expect(
-        fs.existsSync(path.resolve(__dirname, `../${componentName}.ts`)) ||
-          fs.existsSync(path.resolve(__dirname, `../${componentName}.tsx`)),
-      ).toBeTruthy();
-    });
-  });
-
   // make sure that there is a version import in each corresponding top level component file
```

Comment:
Now that the previous test checking that the file exists is removed, what does a failure in the version import test look like if the top-level file doesn't exist? Want to make sure it's not too confusing/misleading.

Reply:
There aren't any visible changes in the top-level version test message in ComponentConformance but it is a bit confusing. I'm planning on re-releasing the component-has-version test after this pull request and it should improve the error message.

Before:
![](https://i.imgur.com/ETmMNgq.png)

After:
![image](https://user-images.githubusercontent.com/7192338/97365863-9f65a600-1863-11eb-9dab-633a1009cc90.png)